x/ccv/consumer/types/keys.go

```diff
@@ -49,6 +49,24 @@ const (
 	// received over CCV channel but not yet flushed over ABCI
 	PendingChangesByteKey
 
+	// PendingDataPacketsByteKey is the byte key for storing
+	// a list of data packets that cannot be sent yet to the provider
+	// chain either because the CCV channel is not established or
+	// because the client is expired
+	PendingDataPacketsByteKey
```

Comment:
`PendingDataPacketsByteKey` is not used. It should be marked as deprecated in favor of  `PendingDataPacketsBytePrefix`, see `PendingDataPacketsKey`.

Reply:
I second this.
Please add `[DEPRECATED]` just above the definition

Reply:
done, see https://github.com/cosmos/interchain-security/pull/991/commits/edcb0aed09b24be9a673750ccc2faf06d58f0a8f